.autover/changes/16cc3c6a-8fa0-410b-ba57-74221abb086a.json

```diff
@@ -0,0 +1,48 @@
+{
+  "Projects": [
+    {
+      "Name": "Amazon.Lambda.RuntimeSupport",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Add support for parameterized logging method with exception to global logger LambdaLogger in Amazon.Lambda.Core"
+      ]
+    },  
+    {
+      "Name": "Amazon.Lambda.Core",
+      "Type": "Minor",
+      "ChangelogMessages": [
+        "Added log level version of the static logging functions on Amazon.Lambda.Core.LambdaLogger"
+      ]
+    }   
+    {
+      "Name": "Amazon.Lambda.AspNetCoreServer",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Update Amazon.Lambda.Logging.AspNetCore dependency"
+      ]
+    },    
+    {
+      "Name": "Amazon.Lambda.AspNetCoreServer.Hosting",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Update Amazon.Lambda.Logging.AspNetCore dependency"
+      ]
+    },  
+    {
+      "Name": "Amazon.Lambda.Logging.AspNetCore",
+      "Type": "Minor",
+      "ChangelogMessages": [
+        "Add support Lambda log levels",
+		"Change build target from .NET Standard 2.0 to .NET 6 and NET 8 to match Amazon.Lambda.AspNetCoreServer"
+      ]
+    },  
+    {
+      "Name": "Amazon.Lambda.TestUtilities",
+      "Type": "Minor",
```

Comment:
I would feel more comfortable making this a Major version

Reply:
Done